### PR TITLE
Fix broken links in README

### PR DESCRIPTION
**Pull request by thaJeztah**
ping @amitshukla @vieux @moxiegirl

### CODE REVIEW

**Comment by moxiegirl** [comment]
LGTM

File: api/README.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
-API](https://docs.docker.com/reference/api/docker_remote_api/).  To read the
-end-user API documentation, visit [the Swarm API documentation on
-docs.docker.com](https://docs.docker.com/swarm/api/swarm-api/). If you want to
-modify the discovery API documentation, start with [the `docs/api/swarm-api.md`
-file](https://github.com/docker/swarm/blob/master/docs/api/swarm-api.md) in this
-project.
+API](https://docs.docker.com/engine/reference/api/docker_remote_api/). To read
+the end-user API documentation, visit [the Swarm API documentation on
+docs.docker.com](https://docs.docker.com/swarm/swarm-api/). If you want to
+modify the discovery API documentation, start with [the `docs/warm-api.md`
```
Comment (dongluochen) [inline api/README.md]: `warm` -> `swarm`

Comment (thaJeztah) [inline api/README.md]: good catch; updated

**Comment by amitshukla** [comment]
LGTM with warm renamed to swarm as Dong points out above.

**Comment by thaJeztah** [comment]
ping @dongluochen ptal

**Comment by dongluochen** [comment]
LGTM. Thanks @thaJeztah!

**Pull request merged**